.github/workflows/pr-realm-js.yml

```diff
@@ -0,0 +1,68 @@
+name: pr-realm-js
+on: pull_request
+
+jobs:
+  build-prebuild:
+    name: Build for Node.js on ${{ matrix.variant.os }} ${{ matrix.variant.arch }}
```

Comment:
Let's remove `Node.js` from the title, as this is actually building for `react-native` and `electron` as well.  We can either add another matrix property, such as {{ matrix.variant.platform }}, or we can just say `Build for ${{ matrix.variant.os}} ${{matrix.variant.arch}}`.
I would go with the latter, unless we find that we are making different builds for different platforms, in which case the former would be more appropriate.